### DISCUSSION TITLE
changing GPIO-Pins in Admin Mode

### DISCUSSION THREAD

**Discussion by ledinator** [forkineye/ESPixelStick#633] [General]
Hi, I programmed my ESP32 board using 4.0-beta4. On my board the CS-Pin of the SD-card shield is connected to the GPIO33 Pin. When Advanced mode is activated I can only change the CS-Pin up to 24 in the web interface. Is there an option to select GPIO Pins higher than 24?

**Comment by MartinMueller2003** [comment]
Yes. This has been changed in the newer builds. GPIOs now go to 36. Use one of the recent builds made in the past 6 months (I would use the one  from last week).
